### ISSUE TITLE
Can't access subscribe page on mobile

### ISSUE THREAD

**Issue by marcelkornblum** [devcon-london/website#14]
Am using brave (powered by chrome) on Pixel 2/Android Q.
Tapping subscribe link doesn't do anything, opening it in a new page results in a GitHub branded 404 page.
This is after GitHub login process appeared to complete successfully (hard to tell as site state didn't update after callback)

**Comment by marcelkornblum** [comment]
I think this is related to the signin not working - I go through the GH screens (and they now flash past as I've already authed the app) but the originating site doesn't update - also on my Chrome/Win desktop. I asked a colleague to try (it's a genuine recommendation to the group) and his browser updated after signin to show a logout link.

**Comment by lusocoding** [comment]
I think the trouble you're having is related to the Brave Shields. If you notice there is some content being blocked when you try to sign in with the GH account. If you disable brave shields for DevCon website you should be fine.

**Comment by marcelkornblum** [comment]
Yeah I tried that. Same was true on my windows Chrome, so I think it must be something else - unless I managed to get it into a state on mobile that it couldn't rescue itself from even when I was on desktop later. I finally worked around it, I think using Safari on mac...

**Comment by grudelsud** [comment]
hey @marcelkornblum we have found and fixed a bug in #59 that might have caused this problem. Would it be possible for you to check again? I've just tested with my Chrome on Android and it seems to work fine. Would be good to know if we can close this bug.

**Comment by silviopaganini** [comment]
is this closed now? @marcelkornblum

**Comment by marcelkornblum** [comment]
Just tried on brave and I think this bug is fixed but the flow is still broken. From a GitHub unauthed session on Android/brave the login pop up appears and disappears immediately. I can now access the subscribe page which just has a toaster message telling me I need to sign in.

**Comment by grudelsud** [comment]
I think someone else mentioned this issue on brave as well. Bloody developers always using obscure browsers... 😂

**Comment by marcelkornblum** [comment]
It's still the Android Chrome under the hood I believe ;)
